torchdata/datapipes/iter/util/unzipper.py

```diff
@@ -19,7 +19,7 @@ class UnZipperIterDataPipe(IterDataPipe[T]):
     r"""
     Takes in a DataPipe of Sequences, unpacks each Sequence, and return the elements in separate DataPipes
     based on their position in the Sequence. The number of instances produced equals to the sequence legnth
-    minus the number of columns to skip.
+    minus the number of columns to skip (functional name: ``unzip``).
```

Comment:
nit: I think you want to move it into the first sentence.

Reply:
Good catch!